src/test_l1_dsDisplay.c

```diff
@@ -339,7 +336,7 @@ void test_l1_dsDisplay_positive_dsGetDisplay(void) {
         snprintf(key_str, sizeof(key_str), "dsDisplay/Video_Ports/%ld", i);
         uint32_t vType = UT_KVP_PROFILE_GET_UINT32(key_str);
```

Comment:
cast to specific type dsVideoPortType_t
check this 
**Review the usage of casting:
Eg: from eIndicator = UT_KVP_PROFILE_GET_UINT32(buffer);
to eIndicator =(dsFPDIndicator_t) UT_KVP_PROFILE_GET_UINT32(buffer);**

Reply:
done